src/Sextant.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.29521.150
+# Visual Studio Version 17
+VisualStudioVersion = 17.0.31919.166
```

Comment:
VS upgraded silently. Is that a problem?

Reply:
Vs upgrade is fine. We are using vs2022 by default now.